soh/soh/resource/type/Skeleton.cpp

```diff
@@ -77,4 +81,45 @@ void SkeletonPatcher::UpdateSkeletons() {
             skel.skelAnime->skeleton = newSkel->skeletonData.skeletonHeader.segment;
     }
 }
+
+void SkeletonPatcher::UpdateTunicSkeletons() {
+    bool isHD = CVarGetInteger("gAltAssets", 0);
```

Comment:
I believe that using "isHD" might not be the most fitting name in this scenario, as the alternative asset feature allows for diverse content. To better match the Cvar assigned, I'd suggest naming the boolean "isAltAssets." This would align the name more closely with the feature's purpose

Reply:
This. Alt Assets were originally called HD Assets at one point in development, but really they serve more purposes than just that, so Alt Assets is the better terminology.

Reply:
Commented earlier but officially adding this to my review. Also it looks like if you switch to checking `vanillaSkeletonPath` and no longer need the child link special case, you won't actually need this bool here at all.

Reply:
changed to isAlt.